src/components/BookList.js

```diff
@@ -0,0 +1,14 @@
+import { useSelector } from 'react-redux';
+import BookCard from './BookCard';
+
+const BookList = () => {
+  const books = useSelector((store) => store.Book.books);
+  return (
+    <div>
+      <BookCard key={books.item_id} books={books} />
+    </div>
+
+  );
+};
+
```

Comment:
- [ ] As suggested by the previous reviewer that kindly include the useEffect directly in this BookList component to make your app follow the project requirements 👍